spec/lib/miq_automation_engine/service_methods/miq_ae_service_generic_object_spec.rb

```diff
@@ -0,0 +1,36 @@
+module MiqAeServiceGenericObjectSpec
+  describe MiqAeMethodService::MiqAeServiceGenericObject do
+    let(:generic_object)  { FactoryGirl.create(:generic_object) }
+    let(:service_go)      { MiqAeMethodService::MiqAeServiceGenericObject.find(generic_object.id) }
+    let(:service)         { FactoryGirl.create(:service) }
+    let(:service_service) { MiqAeMethodService::MiqAeServiceService.find(service.id) }
+
+    context "#add_to_service" do
+      it "adds a generic object to service_resources of a valid service" do
+        service_go.add_to_service(service_service)
+        expect(service.service_resources[0].resource_id).to eq(generic_object.id)
+        expect(service.service_resources[0].resource_type).to eq(generic_object.class.name)
+      end
```

Comment:
@lfu We should also test that adding a second service to the same generic object results in both services being connected.  Ensuring that one service does not get replaced by the other.